### PR TITLE
Add Documentation Vert.x zero config scenario to Common Scenarios

### PR DESCRIPTION
**Pull request by rohanKanojia**
## Description

Add a quick getting started doc for Vert.x to common scenarios section

Related to #1146 

Signed-off-by: Rohan Kumar <rohaan@redhat.com>

## Type of change

 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] Feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change
 - [X] Chore (non-breaking change which doesn't affect codebase;
   test, version modification, documentation, etc.)

## Checklist
 - [ ] I have read the [contributing guidelines](https://www.eclipse.org/jkube/contributing)
 - [ ] I signed-off my commit with a user that has signed the [Eclipse Contributor Agreement](https://www.eclipse.org/legal/ECA.php)
 - [ ] I Added [CHANGELOG](../CHANGELOG.md) entry
 - [ ] I have implemented unit tests to cover my changes
 - [ ] I have updated the [documentation](../kubernetes-maven-plugin/doc) accordingly
 - [ ] No new bugs, code smells, etc. in [SonarCloud](https://sonarcloud.io/dashboard?id=jkubeio_jkube) report
 - [ ] I tested my code in Kubernetes
 - [ ] I tested my code in OpenShift

### CODE REVIEW

**Comment by manusa** [comment]
https://github.com/eclipse/jkube/pull/1172#issuecomment-997789976

**Review by manusa — APPROVED** [review]
LGTM, thx!

File: gradle-plugin/doc/src/main/asciidoc/inc/getting-started/_index.adoc [hunk 1]
```diff
@@ -22,4 +22,6 @@ ifeval::["{task-prefix}" == "oc"]
 
 include::_openshift.adoc[]
 
+include::_vertx.adoc[]
```
Comment (sunix) [inline gradle-plugin/doc/src/main/asciidoc/inc/getting-started/_index.adoc:25]: is it intended that it is in the oc doc only ?

Comment (rohanKanojia) [reply gradle-plugin/doc/src/main/asciidoc/inc/getting-started/_index.adoc:25]: oh, nice catch! this looks wrong. Let me fix it.

File: gradle-plugin/doc/src/main/asciidoc/inc/getting-started/_vertx.adoc [hunk 2]
```diff
@@ -0,0 +1,24 @@
+[[vertx-scenario]]
+== Vert.x
+
+You can easily get started with using {plugin} on an https://vertx.io/[Eclipse Vert.x] without providing any explicit configuration. {plugin} would generate an opinionated container image and manifests by inspecting your project configuration.
```
Comment (sunix) [inline gradle-plugin/doc/src/main/asciidoc/inc/getting-started/_vertx.adoc:4]: maybe refers to https://start.vertx.io/

File: gradle-plugin/doc/src/main/asciidoc/inc/getting-started/_vertx.adoc [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
```
Comment (sunix) [inline gradle-plugin/doc/src/main/asciidoc/inc/getting-started/_vertx.adoc:5]: Maybe add the equivalent of 
https://github.com/eclipse/jkube/blob/c6539d789444f27a74e43e283ffac1891387e754/gradle-plugin/doc/src/main/asciidoc/inc/getting-started/_minikube.adoc?plain=1#L43

> $ ./gradlew quarkusBuild k8sBuild k8sResource k8sApply

**Review by sunix — COMMENTED** [review]
Some suggestions

**Review by sunix — APPROVED** [review]
ok let's merged this one as it is and discuss about other improvements later

**Pull request merged**